README.md

```diff
@@ -16,7 +16,7 @@ a direct visit from the browser.
 
 ```javascript
 // Heroku defines the environment variable PORT, and requires the binding address to be 0.0.0.0
-var host = process.env.PORT ? '0.0.0.0' : '127.0.0.1';
+var host = process.env.HOST || '127.0.0.1';
```

Comment:
I did not want the example to be Heroku-specific, it is just that I first created an application on Heroku, and then open-sourced it (as a result, there are some unnecessary references to Heroku in the documentation).
For local development, it was sufficient to only listen on `127.0.0.1`, for public use on a server it should be listening on `0.0.0.0` instead. It so happens that on heroku the port environment variable is defined, and I used that to listen on a specific port.

If you remove that comment in README.md and server.js, and use `process.env.HOST || '0.0.0.0';` then I will merge the PR.